pulsar-common/src/main/java/org/apache/pulsar/common/naming/TopicName.java

```diff
@@ -67,6 +67,9 @@ public TopicName load(String name) throws Exception {
     public static final TopicName TRANSACTION_COORDINATOR_ASSIGN = TopicName.get(TopicDomain.persistent.value(),
             NamespaceName.SYSTEM_NAMESPACE, "transaction_coordinator_assign");
 
+    public static final TopicName TRANSACTION_COORDINATOR_LOG = TopicName.get(TopicDomain.persistent.value(),
+            NamespaceName.SYSTEM_NAMESPACE, "__transaction_log_");
```

Comment:
are you sure about this '__transaction_log_' ? it is strange that it ends with an underscore character

Reply:
it ends with an underscore character, because there is a coordinator id behind  the `__transaction_log_`.